src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -16,6 +16,15 @@ namespace System.Net.Security
 {
     internal static class SslStreamPal
     {
+        private static bool UseNewCryptoApi = CheckWindowsVersion();
+        private static bool CheckWindowsVersion()
```

Comment:
Nit: the name here seems like it should instead be specific to what it's checking, e.g. Is1809OrNewer, or something like that.

Reply:
updated.